src/lib/components/events/event-item.svelte

```diff
@@ -49,6 +49,8 @@
         <p class="event-location">
           {info.location === 'Discord' || info.location === 'Zoom'
             ? `Hosted on ${info.location}`
+            : info.location == null || info.location.match(/^ *$/) !== null
+            ? 'TBD'
```

Comment:
I think replacing this entire nested ternary with simply `${info.location?.trim() || "TBD"}` is more concise and covers all the edge cases as trim should reduce any spaces to empty strings that evaluate to falsey and a null should short circuit to undefined

Reply:
Do we want Discord and Zoom events to still be displayed as "Hosted on X" tho?

Reply:
I guess we could keep that part? But also why not just put the "Hosted on" in the actual calendar?